### DISCUSSION TITLE
Is there any plan to add tricore 1.6.2 architecture?

### DISCUSSION THREAD

**Discussion by DownyBehind** [NationalSecurityAgency/ghidra#5694] [Q&A]
Hi everyone

as far as I know, TC3xx (AURIX2G) is tricore 1.6.2 architecture. 

So I wonder if is there any plan to add this architecture in Ghidra.

Thank you!

**Comment by rollsch** [comment]
So is TC29x

So far I have no run into any issues decompiling 39x in 29x mode.
